sonar-project.properties

```diff
@@ -1,11 +1,11 @@
 sonar.organization=sap-1
 sonar.projectKey=SAP_open-ux-tools
 sonar.sources=.
-sonar.inclusions=**/*.ts
+sonar.inclusions=**/*.ts, packages/ui-components/src/**/*.tsx
```

Comment:
```suggestion
sonar.inclusions=**/*.ts, **/*.tsx
```
Should this be more general?

Reply:
I can try, but I assume it would start to track coverage for `*.stories.tsx`

Reply:
@devinea 
your proposed variant works as expected - commit 2cdafaa3726aca9ec7803f78e1b8755a4f7da3e2